tests/Integration/Behaviour/Features/Scenario/Product/update_description.feature

```diff
@@ -0,0 +1,40 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s product --tags update-description
+@reset-database-before-feature
+Feature: Update product descriptions from Back Office
+  As a BO user I need to be able to update product description and short description
+
+  @update-description
+  Scenario: Update product descriptions
+    Given I add product "product1" with following basic information:
+      | name       | en-US:potato |
+      | is_virtual | false        |
+    And product "product1" localized "description" should be "en-US:"
+    And product "product1" localized "description_short" should be "en-US:"
+    When I update product "product1" descriptions with following information:
+      | description       | en-US:sweet potato        |
+      | description_short | en-US:Just a sweet potato |
+    Then product "product1" localized "description" should be "en-US:sweet potato"
+    Then product "product1" localized "description_short" should be "en-US:Just a sweet potato"
+
```

Comment:
Could you please add a "Partial update" scenario, to validate that we can udpate only `description` and `short_description` isn't modified (and vice versa)

This is specifically to avoid regression on this partial feature (which can happen quickly with a quick modif without paying attention)

Reply:
modified one of scenarios to cover this.